src/utilities/__tests__/stripIgnoredCharacters-test.js

```diff
@@ -0,0 +1,489 @@
+    expectStripped('" "').toStayTheSame();
+    expectStripped('","').toStayTheSame();
+    expectStripped('",,"').toStayTheSame();
+    expectStripped('",|"').toStayTheSame();
+
+    for (const ignored of ignoredTokens) {
+      expectStripped(JSON.stringify(ignored)).toStayTheSame();
+
+      for (const anotherIgnored of ignoredTokens) {
+        expectStripped(
+          JSON.stringify(ignored + anotherIgnored),
+        ).toStayTheSame();
+      }
+    }
+
+    expectStripped(JSON.stringify(ignoredTokens.join(''))).toStayTheSame();
+  });
+
+  it('does not strip ignored tokens embedded in the block string', () => {
+    expectStripped('""","""').toStayTheSame();
+    expectStripped('""",,"""').toStayTheSame();
+    expectStripped('""",|"""').toStayTheSame();
+
+    const ignoredTokensWithoutFormating = ignoredTokens.filter(
+      token => ['\n', '\r', '\r\n', '\t', ' '].indexOf(token) === -1,
+    );
+    for (const ignored of ignoredTokensWithoutFormating) {
+      expectStripped('"""|' + ignored + '|"""').toStayTheSame();
+
+      for (const anotherIgnored of ignoredTokensWithoutFormating) {
+        expectStripped(
+          '"""|' + ignored + anotherIgnored + '|"""',
+        ).toStayTheSame();
+      }
+    }
+
+    expectStripped(
+      '"""|' + ignoredTokensWithoutFormating.join('') + '|"""',
+    ).toStayTheSame();
+  });
+
+  it('strips ignored characters inside block strings', () => {
+    function expectStrippedString(blockStr) {
+      const originalValue = lexValue(blockStr);
+
+      const strippedStr = stripIgnoredCharacters(blockStr);
+      const strippedValue = lexValue(strippedStr);
+
+      invariant(originalValue === strippedValue);
+      return expectStripped(blockStr);
+    }
+
+    expectStrippedString('""""""').toStayTheSame();
+    expectStrippedString('""" """').toEqual('""""""');
+
+    expectStrippedString('"""a"""').toStayTheSame();
+    expectStrippedString('""" a"""').toEqual('""" a"""');
+    expectStrippedString('""" a """').toEqual('""" a """');
+
+    expectStrippedString('"""\n"""').toEqual('""""""');
+    expectStrippedString('"""a\nb"""').toEqual('"""a\nb"""');
+    expectStrippedString('"""a\rb"""').toEqual('"""a\nb"""');
+    expectStrippedString('"""a\r\nb"""').toEqual('"""a\nb"""');
+    expectStrippedString('"""a\r\n\nb"""').toEqual('"""a\n\nb"""');
+
+    expectStrippedString('"""\\\n"""').toStayTheSame();
+    expectStrippedString('""""\n"""').toStayTheSame();
+    expectStrippedString('"""\\"""\n"""').toEqual('"""\\""""""');
+
+    expectStrippedString('"""\na\n b"""').toStayTheSame();
+    expectStrippedString('"""\n a\n b"""').toEqual('"""a\nb"""');
+    expectStrippedString('"""\na\n b\nc"""').toEqual('"""a\n b\nc"""');
+
+    // Testing with length >5 is taking exponentially more time however it
+    // highly recommended to test with increased limit if you make any change
+    const maxCombinationLenght = 5;
+    const possibleChars = ['\n', ' ', '"', 'a', '\\'];
```

Comment:
This test helped me to uncovered that
```
"""
a
 a
"""
```
can't be converted into:
```
"""a
 a"""
```
Since you will change string value from `a\n a` into `a\na`.
That's why I think it's important to keep it even if can't run it with length `>5` by default.